spark/dl/src/test/scala/com/intel/analytics/bigdl/visualization/SummarySpec.scala

```diff
@@ -91,6 +91,7 @@ class SummarySpec extends FlatSpec with Matchers with BeforeAndAfter {
   }
 
   "read/write event file" should "work properly" in {
+    TestUtils.cancelOnWindows()
```

Comment:
hadoop will try to call chmod underhood and it will cause a failure. So cancel it on windows